### ISSUE TITLE
Vaccination Certificate Showing Wrong Age

### ISSUE THREAD

**Issue by Priyadarshanvijay** [cowinapi/developer.cowin#357]
The certificate generated after getting vaccinated successfully shows the wrong age. It shows the current age + 1. This could be due to the system directly subtracting the `current year` from the `year of birth`. Can we please fix this @cowinapi?

**Comment by karsumit94** [comment]
For the same reason one of my relative had to declare her age as 60 while she was 59.5 yrs old. 45+ had a different queue and 60+ had a different queue. She was in the 45+ queue and had to visit the vaccination center the next day.

**Comment by kunalkankariya** [comment]
@Priyadarshanvijay - This is a very basic mistake. I wonder who is developing the app. Aboslutely ridiculous.

**Comment by parthjawale** [comment]
Yeah, this issue is causing rejection of my COVID certificate as the age is incorrect. This should be corrected on priority please.
